src/main/java/uk/gov/justice/hmpps/prison/repository/ReferenceDataRepository.java

```diff
@@ -75,7 +75,7 @@
                 createParams("domain", domain, "code", code),
                 REF_CODE_DETAIL_ROW_MAPPER);
 
-        final var referenceCodeAsList = convertToReferenceCodes(rcdResults, false);
+        final var referenceCodeAsList = convertToReferenceCodes(rcdResults);
 
```

Comment:
Unrelated warning clean-up; this was always false, so removed with the consequences below